pyaedt/edb_grpc/core/edb_data/hfss_simulation_setup_data.py

```diff
@@ -0,0 +1,2146 @@
+        MeshOperation.__init__(self, parent, mesh_operation)
+
+    @property
+    def skin_depth(self):
+        """Skin depth value.
+
+        Returns
+        -------
+        str
+        """
+        return self.mesh_operation.SkinDepth
+
+    @skin_depth.setter
+    def skin_depth(self, value):
+        self.mesh_operation.SkinDepth = value
+        self._parent._update_setup()
+
+    @property
+    def surface_triangle_length(self):
+        """Surface triangle length value.
+
+        Returns
+        -------
+        str
+        """
+        return self.mesh_operation.SurfTriLength
+
+    @surface_triangle_length.setter
+    def surface_triangle_length(self, value):
+        self.mesh_operation.SurfTriLength = value
+        self._parent._update_setup()
+
+    @property
+    def number_of_layer_elements(self):
+        """Number of layer elements.
+
+        Returns
+        -------
+        str
+        """
+        return self.mesh_operation.NumLayers
+
+    @number_of_layer_elements.setter
+    def number_of_layer_elements(self, value):
+        self.mesh_operation.NumLayers = str(value)
+        self._parent._update_setup()
+
+
+class HfssPortSettings(object):
+    """Manages EDB methods for HFSS port settings."""
+
+    def __init__(self, parent):
+        self._parent = parent
+
+    @property
+    def _hfss_port_settings(self):
+        return self._parent._edb_sim_setup_info.SimulationSettings.HFSSPortSettings
+
+    @property
+    def max_delta_z0(self):
+        """Maximum change to Z0 in successive passes.
+
+        Returns
+        -------
+        float
+        """
+        return self._hfss_port_settings.MaxDeltaZ0
+
+    @max_delta_z0.setter
+    def max_delta_z0(self, value):
+        self._hfss_port_settings.MaxDeltaZ0 = value
+        self._parent._update_setup()
+
+    @property
+    def max_triangles_wave_port(self):
+        """Maximum number of triangles allowed for wave ports.
+
+        Returns
+        -------
+        int
+        """
+        return self._hfss_port_settings.MaxTrianglesWavePort
+
+    @max_triangles_wave_port.setter
+    def max_triangles_wave_port(self, value):
+        self._hfss_port_settings.MaxTrianglesWavePort = value
+        self._parent._update_setup()
+
+    @property
+    def min_triangles_wave_port(self):
+        """Minimum number of triangles allowed for wave ports.
+
+        Returns
+        -------
+        int
+        """
+        return self._hfss_port_settings.MinTrianglesWavePort
+
+    @min_triangles_wave_port.setter
+    def min_triangles_wave_port(self, value):
+        self._hfss_port_settings.MinTrianglesWavePort = value
+        self._parent._update_setup()
+
+    @property
+    def enable_set_triangles_wave_port(self):
+        """Whether to enable setting of minimum and maximum mesh limits for wave ports.
+
+        Returns
+        -------
+        bool
+            ``True`` if triangles wave port  is used, ``False`` otherwise.
```

Comment:
```suggestion
            ``True`` if triangles wave port is used, ``False`` otherwise.
```
This description doesn't seem to match the property description.